recipes/libcoap/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "20200424":
```

Comment:
As already said. Please use the official releases

Creating own releases is a matter of last resort, not something to do because as a consumer we might be impatient 😄

Reply:
Read the comments above 😉

Reply:
I don't read any comments which would be a pro-argument for date based versioning. Last version is from Nov, @obgm said there will be soon-ish a new release